### PR TITLE
Yunyidev

### PR DESCRIPTION
**Pull request by yunyit**
Let's try the following steps this time:

Owner:  After selecting Review Changes, a menu appears on the right. 
- 	Owner: You can add your comments
- 	Owner: In this case, there is something missing before you want to approve the merge. 
- 	Owner: Describe it and select “Request Changes” to send it back to your Collaborator.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # cdh-github-example
 Example repo for the CDH RSE summer school code collaboration workshop. Inspired by the Turing Research Software Engineering workshop example GitHub repo here https://github.com/alan-turing-institute/github-example
 
-From Yunyi
+Request for Review Yunyi
```
Comment (ThaoDo02) [inline README.md:4]: What is this?

**Review by ThaoDo02 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yunyit) [reply README.md:4]: It's a cute lil note :)

**Review by yunyit — COMMENTED** [review]
Revised!

**Review by ThaoDo02 — APPROVED** [review]

**Comment by ThaoDo02** [comment]
Okay done!

**Comment by yunyitang820** [comment]
Completed!

**Comment by yunyitang820** [comment]
Approved the merge here #8

**Pull request merged**